cinema/views.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from django.core.handlers.wsgi import WSGIRequest
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@csrf_exempt
+def movie_list(request: WSGIRequest) -> JsonResponse:
+    print(type(request))
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return JsonResponse(serializer.data, safe=False)
+
+    elif request.method == 'POST':
+        data = JSONParser().parse(request)
+        serializer = MovieSerializer(data=data)
+        if serializer.is_valid():
+            serializer.save()
```

Comment:
you can remove if-statement, for this use serializer.is_valid(raise_exception=True)
Fix in all places